src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,44 @@
+package duke;
+import tasks.TaskList;
```

Comment:
Should there be a line break after the package statement before the import statements?